File: Porting/Porting Envifate/Envifate/src/Library/TestCn.jl
module TestCn

#= imports
	import sqlite3
	import ipdb
=#

import DBInterface as dbi
import SQLite as sql


db = sql.DB("substance.db")

query_cn = sql.Stmt( db, "SELECT * FROM cn" )
results = dbi.execute( query_cn )

#listaclc = Dict()
#for row in results
#	lista_soil = [ x for x in row ]
#	listaclc[ row[5] ] = lista_soil	
#end
listaclc = Dict( row[5] => [ x for x in row ] for row in results )

println(listaclc)

end # module

File: Porting/Porting Envifate/Library/test_cn.py

import sqlite3
import ipdb
conn = sqlite3.connect("substance.db")
cursor=conn.cursor()

listaclc={}

query_cn="select * from cn"
#import pdb; pdb.set_trace()
cursor.execute(query_cn)

sql_fetch=cursor.fetchall()


for row in sql_fetch:
	lista_soil=[]
	for x in row:
		lista_soil.append(x)
	#ipdb.set_trace()
	listaclc[row[5]]=lista_soil	
	#listaclc.append(lista_soil)

conn.close()

print(listaclc)